### DISCUSSION TITLE
Help on Images for Blogs and Cards

### DISCUSSION THREAD

**Discussion by technosprout** [gethyas/hyas#437] [Q&A]
Are there any resources to help with images on blogs post etc.
how to Store images in assets>>images and use them at a particular resize using .resources.Get or storing them under the content>>  folder and  using them with .Resources.Get

**Comment by h-enk** [comment]
Hello, here's the first version of the Hyas Images documentation: https://images.gethyas.com/

It's still under development. Please let me know if something's unclear or is missing — that would help!
